### DISCUSSION TITLE
Make KeystoneConfig `config` object available to custom backend functions

### DISCUSSION THREAD

**Discussion by MurzNN** [keystonejs/keystone#7203] [Feature requests]
Sometimes in custom functions on backend side we need to get some information from Keystone `config` object. The `config` object is only one time passed to `createSystem()` function here: 
https://github.com/keystonejs/keystone/blob/911972098c1bdaa6e64a7aee028b3446e00106e5/packages/keystone/src/lib/createSystem.ts#L64 
after this - to `makeCreateContext()`:
https://github.com/keystonejs/keystone/blob/4d7483d9bbd6a4213f7100e6624367f6a0953b68/packages/keystone/src/lib/context/createContext.ts#L18
and after that - it's not passed to any variable, that is accessible for external code.

So maybe add it to `context` or other variable, that we can import in custom code on backend side?

**Comment by dcousens** [comment]
@MurzNN is there a underlying preference for this instead compared to importing it natively within your own code?
